picard/ui/metadatabox/__init__.py

```diff
@@ -282,19 +285,61 @@ def _copy_value(self):
                 except (TypeError, ValueError) as why:
                     log.warning(why)
                     value = ['']
-
-            if value is not None:
-                self.tagger.clipboard().setText(MULTI_VALUED_JOINER.join(value))
-                self.clipboard = value
+            return value
+
+        #TODO: Support  copying data from multiple files
+
+        items = self.selectedItems()
+        if len(items) > 1:
+            # We have multiple tags selected, so copy them as a JSON string
+            data = {}
+            for item in items:
+                key = self.tag_diff.tag_names[item.row()]
+                if key not in data:
+                    data[key] = {}
+
+                fieldnames = {self.COLUMN_TAG: 'tag_name', self.COLUMN_ORIG: 'original_value', self.COLUMN_NEW: 'new_value'}
```

Comment:
Do we actually need the `original_value` and `new_value` fields? Wouldn't a simpler structure like this work?

```json
{
  "genre": ["value1", "value2"],
  "artist": ["value1"]
}
```

As I see it the pasting code prefers new anyway, and copied valued depend on what was selected.

Btw, what I really like about the implementation is how it is possible to CTRL select some old and some new values and have them copied properly to another file.

Reply:
In terms of copying, I don't think we should add logic to what we "think" the user intends, lets just put everything in the clipboard in case they want to see before and after. If they do only want the new values then they can just select and copy those.